MapboxCoreNavigationTests/DistanceFormatterTests.swift

```diff
@@ -7,74 +7,67 @@ let oneFeet: CLLocationDistance = 0.3048
     var distanceFormatter = DistanceFormatter(approximate: true)
     
     override func setUp() {
         super.setUp()
-        spokenDistanceFormatter.unitStyle = .long
     }
     
-    func assertDistance(_ distance: CLLocationDistance, spoken: String, displayed: String) {
-        let spokenString = spokenDistanceFormatter.string(from: distance)
+    func assertDistance(_ distance: CLLocationDistance, displayed: String) {
         let displayedString = distanceFormatter.string(from: distance)
-        XCTAssert(spokenString.contains(spoken), "Spoken '\(spokenString)' should be equal to '\(spoken)'")
         XCTAssert(displayedString.contains(displayed), "Displayed: '\(displayedString)' should be equal to \(displayed)")
     }
     
     func testDistanceFormatters_US() {
-        spokenDistanceFormatter.numberFormatter.locale = Locale(identifier: "en-US")
         distanceFormatter.numberFormatter.locale = Locale(identifier: "en-US")
         
-        assertDistance(0,               spoken: "0 feet",               displayed: "0 ft")
-        assertDistance(oneFeet*50,      spoken: "50 feet",              displayed: "50 ft")
-        assertDistance(oneFeet*100,     spoken: "100 feet",             displayed: "100 ft")
-        assertDistance(oneFeet*249,     spoken: "250 feet",             displayed: "250 ft")
-        assertDistance(oneFeet*305,     spoken: "300 feet",             displayed: "300 ft")
-        assertDistance(oneFeet*997,     spoken: "1,000 feet",           displayed: "1,000 ft")
-        assertDistance(oneMile*0.25,    spoken: "a quarter mile",       displayed: "0.2 mi")
-        assertDistance(oneMile/2,       spoken: "a half mile",          displayed: "0.5 mi")
-        assertDistance(oneMile*0.75,    spoken: "3 quarters of a mile", displayed: "0.8 mi")
-        assertDistance(oneMile,         spoken: "1 mile",               displayed: "1 mi")
-        assertDistance(oneMile*2.5,     spoken: "2 & a half miles",     displayed: "2.5 mi")
-        assertDistance(oneMile*2.9,     spoken: "2.9 miles",            displayed: "2.9 mi")
-        assertDistance(oneMile*3,       spoken: "3 miles",              displayed: "3 mi")
-        assertDistance(oneMile*3.5,     spoken: "4 miles",              displayed: "4 mi")
-        assertDistance(oneMile*5.4,     spoken: "5 miles",              displayed: "5 mi")
+        assertDistance(0,               displayed: "0 ft")
+        assertDistance(oneFeet*50,      displayed: "50 ft")
+        assertDistance(oneFeet*100,     displayed: "100 ft")
+        assertDistance(oneFeet*249,     displayed: "250 ft")
+        assertDistance(oneFeet*305,     displayed: "300 ft")
+        assertDistance(oneMile*0.1,     displayed: "0.1 mi")
+        assertDistance(oneMile*0.25,    displayed: "0.2 mi")
+        assertDistance(oneMile/2,       displayed: "0.5 mi")
+        assertDistance(oneMile*0.75,    displayed: "0.8 mi")
```

Comment:
Shouldn't this round the same as `oneMile*0.25`? We should be showing every `0.1` mile increment under 3 miles.

Reply:
Seems to be a floating point issue. 0.251 rounds to 0.3.